rest-api-backend/main.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/gorilla/mux"
+)
+
+// Users struct contains 2 properies that represents all the users on the site
+type Users struct {
+	Userid   string `json:"userid"`
+	Username string `json:"username"`
+}
+
+var db *sql.DB
+var err error
+
+func main() {
+
+	// connecting to the database
+	db, err = sql.Open("mysql", "root:@/lottery")
+
+	if err != nil {
+		panic(err.Error())
```

Comment:
Don't panic.

https://golang.org/doc/effective_go.html#errors

https://medium.com/@hussachai/error-handling-in-go-a-quick-opinionated-guide-9199dd7c7f76